lib/SparkPost/SparkPost.php

```diff
@@ -64,11 +65,16 @@ public function setupUnwrapped ($endpoint) {
    * Merges passed in headers with default headers for http requests
    */
   public function getHttpHeaders() {
+
     $defaultOptions = [
         'Authorization' => $this->config['key'],
         'Content-Type' => 'application/json',
     ];
 
+    if( $this->config['subaccount_id'] ){
```

Comment:
I think this should probably be renamed to `default_subaccount`. Setting this as part of the config means that anytime you want to change subaccounts, you'll have to ether update the config with `setConfig` or instantiate a new SparkPost Object.